### PR TITLE
Cherry-Pick Fix Node Autocomplete Regressions

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Cherry-Pick Fix Node Autocomplete Regressions
https://github.com/DynamoDS/Dynamo/pull/14275

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

Cherry-Pick Fix Node Autocomplete Regressions

### Reviewers

@QilongTang @reddyashish

### CODE REVIEW

File: test/DynamoCoreWpfTests/SearchSideEffects.cs [hunk 1]
```diff
@@ -22,25 +22,24 @@ public void WhenStartingDynamoInputAndOutputNodesAreNolongerMissingFromSearch()
             Assert.AreEqual(1, ViewModel.CurrentSpaceViewModel.InCanvasSearchViewModel.FilteredResults.Count(x => x.Model.Name == "Output"));
         }
 
-
         /// <summary>
         /// This test will validate that the nodes "Input", "Output", "And", "Or", "Not", "+", "-"  appear in the InCanvasSearch results
         /// </summary>
         [Test]
-        [Category("UnitTests")]
```
Comment (reddyashish) [inline test/DynamoCoreWpfTests/SearchSideEffects.cs:30]: @RobertGlobant20 [Category("UnitTests")] should also be present.

Comment (RobertGlobant20) [reply test/DynamoCoreWpfTests/SearchSideEffects.cs:30]: I searched for Failure category in Dynamo solution and in all the places is only Failure, do you have an example of a test with both categories?
e.g.
image

Comment (reddyashish) [reply test/DynamoCoreWpfTests/SearchSideEffects.cs:30]: You can just add another category attribute like this: https://github.com/DynamoDS/Dynamo/blob/master/test/DynamoCoreTests/UnitsOfMeasureTests.cs#L1047

Comment (RobertGlobant20) [reply test/DynamoCoreWpfTests/SearchSideEffects.cs:30]: @reddyashish ok got your point.
Due that this PR is a cherry-pick, could you approve and merge it? (If I do changes over this branch won't be cherry-picking the commit).
I will do the change in the original branch and then create another cherry-pick PR.

Comment (QilongTang) [reply test/DynamoCoreWpfTests/SearchSideEffects.cs:30]: This also works

**Review by QilongTang — APPROVED** [review]

**Pull request merged**